samples/drivers/memc/boards/apollo4p_evb.overlay

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2024, Ambiq Micro Inc. <www.ambiq.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	aliases {
+		psram0 = &aps256n;
+	};
+};
+
+&gpio64_95 {
+	status = "okay";
+};
+
+&gpio32_63 {
+	status = "okay";
+};
+
+&mspi0 {
+
+	pinctrl-0 = <&mspi0_default>;
+	pinctrl-1 = <&mspi0_sleep>;
+	pinctrl-2 = <&mspi0_psram>;
+	pinctrl-names = "default","sleep","psram";
+	status = "okay";
+
+	ce-gpios = <&gpio32_63 57 GPIO_ACTIVE_LOW>;
+
+	cmdq-buffer-location = ".mspi_buff";
+	cmdq-buffer-size = <256>;
+
+	ambiq,rx-config = <2 0 0 0 1 0 0 0 0 0>;
+
```

Comment:
```suggestion
	dqs-support;
```
This should be set to indicate the controller does support DQS feature.
In the controller driver, the mspi-dqs-enable setting should be checked against this one.